{{cookiecutter.example_name}}/Code.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+
+# Copyright Insight Software Consortium
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0.txt
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import itk
+
+if len(sys.argv) != 3:
+    print("Usage: " + sys.argv[0] + "<InputFileName> "
+          "<OutputFileName>")
+    sys.exit(1)
+
+input_file_name = sys.argv[1]
+output_file_name = sys.argv[2]
+
+Dimension = 2
+PixelType = itk.UC
+ImageType = itk.Image[PixelType, Dimension]
+
+input_image = itk.imread(input_file_name, PixelType)
+
+{{ cookiecutter.filter_instance }} = itk.{{ cookiecutter.class_name }}[ImageType, ImageType].New()
```

Comment:
Should we let the filter infer the image type, set parameters as positional arguments to the class name and do not create an instance with `New` as in the `ITK 5.0 Beta 1: Pythonic Interface` announcement?

Reply:
Inferred pixel type is preferred.

Reply:
Done in 5409bf6.